include/foonathan/memory/memory_pool.hpp

```diff
@@ -54,6 +54,20 @@ namespace foonathan
 
             static constexpr std::size_t min_node_size =
                 FOONATHAN_IMPL_DEFINED(free_list::min_element_size);
+                
+            /// \returns The minimum block size required for certain number of \concept{concept_node,node}.
+            /// \requires \c node_size must be a valid \concept{concept_node,node size}
+            /// and \c number_of_nodes must be a non-zero value.
+            static constexpr std::size_t min_block_size(std::size_t node_size, std::size_t number_of_nodes)
+            {
+                return
+                    detail::memory_block_stack::implementation_offset +
+                    number_of_nodes *
+                    (
+                        ( (node_size > min_node_size) ? node_size : min_node_size) +
```

Comment:
Could you use ifs instead of ternary?

Reply:
Done. I also added a macro so users could check for the availability of the new API